package.json

```diff
@@ -33,5 +37,7 @@
     "broccoli-test-builder": "^0.2.0",
     "testem": "^0.9.0-1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "broccoli-funnel": "^1.0.0"
```

Comment:
Hmm, I'll try. We don't need to watch the files, which is at least half of broccoli-funnel's job